src/data/nat/prime.lean

```diff
@@ -1150,6 +1162,15 @@ end
 lemma mem_factors_mul_right {p a b : ℕ} (hpb : p ∈ b.factors) (ha : a ≠ 0) : p ∈ (a*b).factors :=
 by { rw mul_comm, exact mem_factors_mul_left hpb ha }
 
+lemma eq_two_pow_or_exists_odd_prime_and_dvd (n : ℕ) :
```

Comment:
```suggestion
lemma eq_two_pow_or_exists_odd_prime_dvd (n : ℕ) :
```
Probably this is bikeshedding, but we don't usually put `and` in the name when there are multiple facts.

Reply:
We do have `{nat,pnat,int}.exists_prime_and_dvd` already, though. I think it probably makes more sense to match those.